keps/sig-scheduling/268-priority-preemption/README.md

```diff
@@ -60,6 +67,130 @@ caused by this change.
 * Adequate documentation exists for the features.
 * Test coverage of the features is acceptable.
 
+## Production Readiness Review Questionnaire
```

Comment:
I added the proposal to PRR section as we decided to go with [PRR updates ](https://github.com/kubernetes/kubernetes/issues/99205#issuecomment-788883685). Let me know, if you folks want a new KEP

Reply:
I think what @soltysh was asking is that we have a new KEP that includes the PRR questionnaire

Reply:
@ravisantoshgudimetla pinged me about it, I only cared about it being written down. Whether a new KEP or update to the current I'll leave that decision to both of you :wink: Just make sure to update `kep.yaml` I see it's outdated and gives wrong information about the state of this feature.